client/post-editor/editor-confirmation-sidebar/index.jsx

```diff
@@ -28,6 +29,8 @@ class EditorConfirmationSidebar extends React.Component {
 		onPublish: React.PropTypes.func,
 		post: React.PropTypes.object,
 		savedPost: React.PropTypes.object,
+		privatePost: React.PropTypes.bool,
+		privatePostPasswordValid: React.PropTypes.bool,
```

Comment:
The naming of these props is a bit confusing. Consider `isPrivatePost` and `isPrivatePostPasswordValid`.